ui/article-renderer/src/main/java/org/cru/godtools/article/activity/ArticlesActivity.java

```diff
@@ -82,4 +98,17 @@ private void loadInitialFragmentIfNeeded() {
                 .replace(R.id.frame, ArticlesFragment.newInstance(mTool, mLocale, mCategoryId), TAG_MAIN_FRAGMENT)
                 .commit();
     }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                onBackPressed();
```

Comment:
this breaks the up navigation paradigm

Reply:
@frett I did this because the back button in the action bar wasn't functioning.  Is there a call in your baseToolsFragment that will resolve this.

Reply:
that's because it needed the parent activity defined in the android manifest. I did that in this PR: https://github.com/CruGlobal/godtools-android/pull/502

Reply:
I am going to assume that this will be handle if I use a separate Activity.

Reply:
Ok